### PR TITLE
fix(2734): Builds cannot be started if a pipeline has more than 5 invalid admins

### PR DESCRIPTION
**Pull request by ibu1224**
## Context

Improve the following Issue.
https://github.com/screwdriver-cd/screwdriver/issues/2734

## Objective

In this PR, change the Circuit Breaker so that it is not Open even if a 4XX response code is returned from the GitHub

## References

https://github.com/screwdriver-cd/screwdriver/issues/2734

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -238,6 +238,19 @@ class GithubScm extends Scm {
             this.octokitConfig.baseUrl = `${this.config.gheProtocol}://${this.config.gheHost}/api/v3`;
         }
 
+        let breakerOption = this.config.fusebox.breaker;
```
Comment (itleigns) [inline index.js:241]: Where is this variable used?

File: index.js [hunk 2]
```diff
@@ -238,12 +238,25 @@ class GithubScm extends Scm {
...
+
+        if (breakerOption) {
+            breakerOption.errorFn = function(err) {
+                return !(err.statusCode >= 400 && err.statusCode < 500);
```
Comment (itleigns) [inline index.js:245]: Is it possible that the `statusCode` is `undefined`? If so, you should separate the cases.

File: index.js [hunk 3]
```diff
@@ -238,12 +238,25 @@ class GithubScm extends Scm {
...
+                }
+            };
+        }
         // eslint-disable-next-line no-underscore-dangle
         this.breaker = new Breaker(this._githubCommand.bind(this), {
             // Do not retry when there is a 4XX error
             shouldRetry: err => err && err.statusCode && !(err.statusCode >= 400 && err.statusCode < 500),
             retry: this.config.fusebox.retry,
-            breaker: this.config.fusebox.breaker
+            breaker: breakerOption
```
Comment (y-oksaku) [inline index.js:259]: I think we can make it simpler.

```javascript
breaker: {
    ...this.config.fusebox.breaker,
    errorFn(err) { ... },
}
```

File: index.js (hunk 2 above)
Comment (ibu1224) [reply index.js:245]: I'll check err.statusCode just to be sure.

**Review by y-oksaku — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**